web/index.html

```diff
@@ -34,6 +34,17 @@
   </style>
   <link rel="manifest" href="manifest.json">
   <script>
+    let query = window.location.search;
+    if (query && query.indexOf('renderer=') !== -1) {
+      if (query.indexOf('renderer=html') !== -1) {
+        console.log('HTML renderer requested in the URL');
+        window.flutterWebRenderer = 'html';
+      } else {
```

Comment:
I think this `else` should instead be an explicit check for `renderer=canvaskit` here (similar to the `html` above), otherwise forcing "canvaskit" is going to break "auto" mode in mobile iOS (when hitting gallery.flutter.dev)

Reply:
Ahhh no, this is checking for renderer=anything in 38, nvm!